outputs/gcp.go

```diff
@@ -0,0 +1,77 @@
+package outputs
+
+import (
+	"cloud.google.com/go/pubsub"
+	"context"
+	"encoding/base64"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+	"log"
+	"os"
+	"errors"
+	"encoding/json"
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// NewGCPClient returns a new output.Client for accessing the GCP API.
+func NewGCPClient(config *types.Configuration, stats *types.Statistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+
+	if config.GCP.Credentials != "" {
```

Comment:
why do you set env vars?

Reply:
I had followed the style in the AWS output, but I thought it was odd. I've removed this.